index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your project's title to a meaningful title name(e.g: "Hello Microverse") to make your project more professional and for good accessibility.